spec/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos', bio: 'Teacher from Mexico.')
+  post = Post.new(author: user, title: 'post1', text: 'This is my first Post')
+
+  before { post.save }
+
```

Comment:
Kindly make sure to add test for the validation 'Title must not exceed 250 characters.'  and also for Comments and Likes counters of the post.

Reply:
@Danieldotcomcoder Can you please explain more which test is failing

I am getting all tests passed. See below:

![image](https://github.com/dheerajsachdeva/blog-app/assets/118332295/9b8602f0-19c4-4980-b695-8c3ad53f2295)

Reply:
@Danieldotcomcoder Test for the validation 'Title must not exceed 250 characters.' and also for Comments and Likes counters of the post added